news/4908.bugfix

```diff
@@ -0,0 +1 @@
+FIXED UnboundLocalError by changing local variable 'base_url' which was not referenced before assignment to 'base'.
```

Comment:
Sorry to be nit-picky but the word FIXED is redundant here and mentioning the exact error message caused by the typing/copy-paste error seems to be too much detail being included in a news files.

Reply:
Np :)
Is,
"Replaced typo mistake in subversion support" fine?

Reply:
Yeps. :)